src/lib.rs

```diff
@@ -30,5 +30,6 @@ mod tests;
 mod bench;
 
 pub use derivation::{DerivationError, DerivationIndex, DerivationScheme};
-pub use key::{PrivateKeyError, PublicKeyError, XPrv, XPub, XPRV_SIZE, XPUB_SIZE};
+pub use key::{PrivateKeyError, PublicKeyError, XPrv, XPub, XPRV_SIZE, XPUB_SIZE, CHAIN_CODE_SIZE, EXTENDED_SECRET_KEY_SIZE};
```

Comment:
We can't modify the original lib
```suggestion
pub use key::{PrivateKeyError, PublicKeyError, XPrv, XPub, XPRV_SIZE, XPUB_SIZE};
```

Reply:
can you explain why? I can understand why would try to avoid it, but I don't see this as an issue?

Reply:
Because we want to always keep the original library as it is, the author has absolute freedom about his library. He can modify and update it without affecting our own code. Our code, at the end of the day, is just a wrapper to the original library.

Reply:
I've made this change, although I disagree with the rationale; this is our fork to do as we need, and I see the alternative to this puts us in a worse position.